frontend/app/view/search/search.tsx

```
@@ -0,0 +1,536 @@
+            await createBlockSplitVertically(blockDef, blockId, "after");
+        }
+    };
+
+    const renderHighlight = (line: string) => {
+        if (!query) return <span>{line}</span>;
+        try {
+            let regex: RegExp;
+            if (useRegex) {
+                regex = new RegExp(query, matchCase ? "g" : "gi");
+            } else {
+                const escaped = query.replace(/[-\/\\^$*+?.()|[\]{}]/g, "\\$&");
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)